09:28:03.715 - 37: org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.callAnalysisComponentProcess(417): GRAVE: Exception occurred
org.apache.uima.analysis_engine.AnalysisEngineProcessException: Annotator processing failed.    
	at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.callAnalysisComponentProcess(PrimitiveAnalysisEngine_impl.java:401)
	at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.processAndOutputNewCASes(PrimitiveAnalysisEngine_impl.java:308)
	at org.apache.uima.analysis_engine.asb.impl.ASB_impl$AggregateCasIterator.processUntilNextOutputCas(ASB_impl.java:570)
	at org.apache.uima.analysis_engine.asb.impl.ASB_impl$AggregateCasIterator.<init>(ASB_impl.java:412)
	at org.apache.uima.analysis_engine.asb.impl.ASB_impl.process(ASB_impl.java:344)
	at org.apache.uima.analysis_engine.impl.AggregateAnalysisEngine_impl.processAndOutputNewCASes(AggregateAnalysisEngine_impl.java:265)
	at org.apache.uima.analysis_engine.impl.AnalysisEngineImplBase.process(AnalysisEngineImplBase.java:269)
	at org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.processNext(ProcessingUnit.java:893)
	at org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.run(ProcessingUnit.java:575)
Caused by: org.apache.uima.cas.CASRuntimeException: JCas type "defecto.NoDetector" used in Java code,  but was not declared in the XML type descriptor.
	at org.apache.uima.jcas.impl.JCasImpl.getTypeInit(JCasImpl.java:435)
	at org.apache.uima.jcas.impl.JCasImpl.getType(JCasImpl.java:408)
	at org.apache.uima.jcas.impl.JCasImpl.getCasType(JCasImpl.java:457)
	at org.apache.uima.jcas.impl.JCasImpl.getAnnotationIndex(JCasImpl.java:1569)
	at annotators.NegatedSentenceAnnotator.process(NegatedSentenceAnnotator.java:43)
	at org.apache.uima.analysis_component.JCasAnnotator_ImplBase.process(JCasAnnotator_ImplBase.java:48)
	at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.callAnalysisComponentProcess(PrimitiveAnalysisEngine_impl.java:385)
	... 8 more

09:28:03.747 - 37: org.apache.uima.analysis_engine.impl.AggregateAnalysisEngine_impl.processAndOutputNewCASes(273): GRAVE: Exception occurred
org.apache.uima.analysis_engine.AnalysisEngineProcessException: Annotator processing failed.    
	at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.callAnalysisComponentProcess(PrimitiveAnalysisEngine_impl.java:401)
	at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.processAndOutputNewCASes(PrimitiveAnalysisEngine_impl.java:308)
	at org.apache.uima.analysis_engine.asb.impl.ASB_impl$AggregateCasIterator.processUntilNextOutputCas(ASB_impl.java:570)
	at org.apache.uima.analysis_engine.asb.impl.ASB_impl$AggregateCasIterator.<init>(ASB_impl.java:412)
	at org.apache.uima.analysis_engine.asb.impl.ASB_impl.process(ASB_impl.java:344)
	at org.apache.uima.analysis_engine.impl.AggregateAnalysisEngine_impl.processAndOutputNewCASes(AggregateAnalysisEngine_impl.java:265)
	at org.apache.uima.analysis_engine.impl.AnalysisEngineImplBase.process(AnalysisEngineImplBase.java:269)
	at org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.processNext(ProcessingUnit.java:893)
	at org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.run(ProcessingUnit.java:575)
Caused by: org.apache.uima.cas.CASRuntimeException: JCas type "defecto.NoDetector" used in Java code,  but was not declared in the XML type descriptor.
	at org.apache.uima.jcas.impl.JCasImpl.getTypeInit(JCasImpl.java:435)
	at org.apache.uima.jcas.impl.JCasImpl.getType(JCasImpl.java:408)
	at org.apache.uima.jcas.impl.JCasImpl.getCasType(JCasImpl.java:457)
	at org.apache.uima.jcas.impl.JCasImpl.getAnnotationIndex(JCasImpl.java:1569)
	at annotators.NegatedSentenceAnnotator.process(NegatedSentenceAnnotator.java:43)
	at org.apache.uima.analysis_component.JCasAnnotator_ImplBase.process(JCasAnnotator_ImplBase.java:48)
	at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.callAnalysisComponentProcess(PrimitiveAnalysisEngine_impl.java:385)
	... 8 more

09:28:03.763 - 37: org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.process: GRAVE: The container DocumentAnalyzerAE returned the following error message: Annotator processing failed.     (Thread Name: [CasConsumer Pipeline Thread]::) 
09:28:03.763 - 37: org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.maybeLogSevereException(2498): GRAVE: Thread: [CasConsumer Pipeline Thread]::, message: Annotator processing failed.    
org.apache.uima.analysis_engine.AnalysisEngineProcessException: Annotator processing failed.    
	at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.callAnalysisComponentProcess(PrimitiveAnalysisEngine_impl.java:401)
	at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.processAndOutputNewCASes(PrimitiveAnalysisEngine_impl.java:308)
	at org.apache.uima.analysis_engine.asb.impl.ASB_impl$AggregateCasIterator.processUntilNextOutputCas(ASB_impl.java:570)
	at org.apache.uima.analysis_engine.asb.impl.ASB_impl$AggregateCasIterator.<init>(ASB_impl.java:412)
	at org.apache.uima.analysis_engine.asb.impl.ASB_impl.process(ASB_impl.java:344)
	at org.apache.uima.analysis_engine.impl.AggregateAnalysisEngine_impl.processAndOutputNewCASes(AggregateAnalysisEngine_impl.java:265)
	at org.apache.uima.analysis_engine.impl.AnalysisEngineImplBase.process(AnalysisEngineImplBase.java:269)
	at org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.processNext(ProcessingUnit.java:893)
	at org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.run(ProcessingUnit.java:575)
Caused by: org.apache.uima.cas.CASRuntimeException: JCas type "defecto.NoDetector" used in Java code,  but was not declared in the XML type descriptor.
	at org.apache.uima.jcas.impl.JCasImpl.getTypeInit(JCasImpl.java:435)
	at org.apache.uima.jcas.impl.JCasImpl.getType(JCasImpl.java:408)
	at org.apache.uima.jcas.impl.JCasImpl.getCasType(JCasImpl.java:457)
	at org.apache.uima.jcas.impl.JCasImpl.getAnnotationIndex(JCasImpl.java:1569)
	at annotators.NegatedSentenceAnnotator.process(NegatedSentenceAnnotator.java:43)
	at org.apache.uima.analysis_component.JCasAnnotator_ImplBase.process(JCasAnnotator_ImplBase.java:48)
	at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.callAnalysisComponentProcess(PrimitiveAnalysisEngine_impl.java:385)
	... 8 more

09:28:19.108 - 37: org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.process: ADVERTENCIA: The CAS is being dropped due to the CPM error handling configuration. (Thread Name: [CasConsumer Pipeline Thread]::) Container Name: DocumentAnalyzerAE Cas Processor: org.apache.uima.analysis_engine.impl.AggregateAnalysisEngine_impl
09:28:19.108 - 37: org.apache.uima.collection.impl.cpm.container.ProcessingContainer_Impl.process: GRAVE: The CPM stopped because the configured error threshold 0 was exceeded. (Thread Name: [CasConsumer Pipeline Thread]::) Component Name: DocumentAnalyzerAE 
09:28:19.108 - 37: org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.process: GRAVE: The CPM is terminating. The current component is DocumentAnalyzerAE. (Thread Name: [CasConsumer Pipeline Thread]::) 
09:28:19.124 - 37: org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.process: ADVERTENCIA: The CPM cannot be stopped by force. The current container is DocumentAnalyzerAE. (Thread Name: [CasConsumer Pipeline Thread]::) Reason: The CAS processor DocumentAnalyzerAE is configured to stop the CPM when excessive errors are encountered.  (Thread Name: [CasConsumer Pipeline Thread]::)
09:28:19.124 - 37: org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.maybeLogSevereException(2498): GRAVE: Thread: [CasConsumer Pipeline Thread]::, message: 
org.apache.uima.collection.base_cpm.AbortCPMException: 
	at org.apache.uima.collection.impl.cpm.container.ProcessingContainer_Impl.incrementCasProcessorErrors(ProcessingContainer_Impl.java:795)
	at org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.processNext(ProcessingUnit.java:1037)
	at org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.run(ProcessingUnit.java:575)

09:28:19.124 - 37: org.apache.uima.collection.impl.cpm.engine.CPMEngine.process: INFORMACIÓN: The collection reader thread state is: 1004 (Thread Name: [CasConsumer Pipeline Thread]::) 
09:28:19.124 - 37: org.apache.uima.collection.impl.cpm.engine.CPMEngine.process: INFORMACIÓN: The CPM processing unit is 0 and processing state 2002. (Thread Name: [CasConsumer Pipeline Thread]::)
09:28:19.124 - 37: org.apache.uima.collection.impl.cpm.engine.CPMEngine.process: INFORMACIÓN: The CAS consumer thread state is 2003. (Thread Name: [CasConsumer Pipeline Thread]::) 
09:28:19.124 - 37: org.apache.uima.collection.impl.cpm.engine.CPMEngine.process: INFORMACIÓN: The application stopped the CPM. (Thread Name: [CasConsumer Pipeline Thread]::)
09:28:19.124 - 37: org.apache.uima.collection.impl.cpm.engine.CPMEngine.process: INFORMACIÓN: The CPM engine is stopping. An end-of-file token is added to the worker queue. (Thread Name: [CasConsumer Pipeline Thread]::) Forced stop: true
